files/migrations/0003_update_file_fields.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
```

Comment:
Need to split the field update into two migrations as Django will not be able to detect we are updating field name if we change the field name and verbose_name at the same time